script/bootstrap

```diff
@@ -1,16 +1,16 @@
-#!/bin/sh
+#!/bin/bash
 
 # script/bootstrap: Resolve all dependencies that the application requires to
 #                   run.
 
 set -e
 
-cd "$(dirname "$0")/.."
+cd "${0%/*}/.."
 
 if [ -f "Brewfile" ] && [ "$(uname -s)" = "Darwin" ]; then
   brew update
 
-  brew bundle check 2>&1 >/dev/null || {
+  brew bundle check >/dev/null 2>&1 || {
```

Comment:
Since `&>/dev/null` does the same task, I chose `>/dev/null 2>&1` for readability.